lib/ui/painting.dart

```diff
@@ -860,6 +860,54 @@ class Path extends NativeFieldWrapperClass2 {
   void _arcTo(double left, double top, double right, double bottom,
               double startAngle, double sweepAngle, bool forceMoveTo) native "Path_arcTo";
 
+  /// Appends up to four conic curves weighted to describe an oval of radius
+  /// [radiusX], [radiusY] and rotated by [xAxisRotation].
+  ///
+  /// The first curve begins from the last point in the path and the last ends
+  /// at [arcEndX] and [arcEndY]. The curves follow a path in a direction
+  /// determined by [isClockwiseDirection] and [isLargeArc] in such a way that
+  /// the sweep angle is always less than 360 degrees.
+  ///
+  /// A simple line is appended if either either radii are zero or the last
+  /// point in the path is [arcEndX] and [arcEndY]. The radii are scaled to fit
+  /// the last path point if both are greater than zero but too small to
+  /// describe an arc.
+  void arcToPoint(double radiusX, double radiusY, double xAxisRotation,
```

Comment:
For this to read right, you want the first argument to be the target point. Then the remaining arguments can be named.

So this would be:
```dart
  void arcToPoint(Offset end, { Radius radius: Radius.zero, double angle: 0.0, bool largeArc: false, bool clockwise: false }) { ... }
```

Reply:
The `arcTo` variant above does not make the parameters named. I wanted to be consistent with the other methods in this file. Should I still update? I'll update the signature to put the end point first however.

Reply:
For the other variant, it was a mistake to not make them named, IMHO.

Reply:
(especially the bool argument, that contradicts our style guide)